src/config/index.js

```diff
@@ -100,6 +100,14 @@ const config = {
         revalidationInterval: parseIntEnv('COMPLIANCE_REVALIDATION_INVERVAL', 60 * 60) // 1 hour
     },
 
+    configManager: {
+        impl: env.CONFIG_MANAGER_IMPL,
+        host: env.CONFIG_MANAGER_HOST || 'http://config-manager-api.config-manager-ci.svc.cluster.local:8000',
```

Comment:
port 8081 in CI

Reply:
@Jason-RH does the rest of the hostname look ok for talking too config-manager?

Reply:
@aleccohan Looks like it's actually config-manager-service instead of config-manager-api, but looks good otherwise

Reply:
I may update that later to be more consistent. I'll put up a PR against remediations when I do

Reply:
sounds great!  just pushed up those changes